donut.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <math.h>
+#include <stdint.h>
+#include <string.h>
+#include <unistd.h>
+
+int main() {
+    float A = 0, B = 0;
+    float i, j;
+    int k;
+    float z[1760];
```

Comment:
The variable `z` seems to be used for depth calculation. Consider adding a comment explaining its purpose.